CONTRIBUTING.md

```diff
@@ -82,17 +130,16 @@ Must be one of the following:
 - **feat**: A new feature
 - **fix**: A bug fix
 - **docs**: Documentation only changes
-- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-  semicolons, etc.)
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semicolons, etc.)
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **perf**: A code change that improves performance
 - **test**: Adding missing or correcting existing tests
-- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-  generation
+- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+- **types**: A code change related with TypeScript type
```

Comment:
When making below commit, @Tkdodo used `types` commit type

https://github.com/TanStack/query/commit/39b2f81bc782a8633bd601b6d2e35d2a8c2ec4c0

Reply:
I think I just did that wrong:

https://www.conventionalcommits.org/en/v1.0.0/

> types other than fix: and feat: are allowed, for example [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional) (based on the [Angular convention](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines)) recommends build:, chore:, ci:, docs:, style:, refactor:, perf:, test:, and others.

Reply:
I removed it now